yarn.lock

```diff
@@ -10765,6 +10757,16 @@ query-string@^7.0.0:
     split-on-first "^1.0.0"
     strict-uri-encode "^2.0.0"
 
+query-string@^7.0.1:
```

Comment:
So, `yarn` has this really annoying bug where it sometimes includes multiple versions of a dependency unnecessarily – see how here we have  `7.0.0` and `7.0.1` included in the bundle where just `7.0.1` would be fine as they have the same interface?

Thankfully there is a utility to fix this, I should automate it running after every yarn install to be honest – run this locally on the branch:

`npx yarn-deduplicate yarn.lock`

Reply:
Done